pootle/core/search/base.py

```diff
@@ -16,6 +16,8 @@
 class SearchBackend(object):
     def __init__(self, config_name=None, **kwargs):
         self._setup_settings(config_name)
+        self.weight = 1
```

Comment:
Despite we fallback to 1 we are not telling that anywhere since I feel that it is preferable to have complete settings that explicitly state it. Thus the check.